components/contribution/MergePanel.tsx

```diff
@@ -0,0 +1,191 @@
+    (SelectedBuilding | undefined)[] | null
+  >(null);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setIsLoading(true);
+      const responses = await Promise.all(
+        candidatesToMerge.map((candidate) => fetchBuilding(candidate)),
+      );
+      setCandidatesWithAddresses(responses);
+      setIsLoading(false);
+    };
+    if (candidatesToMerge.length) {
+      fetchData();
+    } else setCandidatesWithAddresses(null);
+  }, [candidatesToMerge]);
+  const cancelMerge = () => {
+    dispatch(Actions.edition.setOperation(null));
+    dispatch(Actions.edition.resetCandidates());
+  };
+  const newMerge = () => {
+    dispatch(Actions.edition.resetCandidates());
+    dispatch(Actions.map.removeBuildings());
+    setNewBuilding(null);
+    dispatch(Actions.map.reloadBuildings());
+  };
+  const selectCandidateToRemove = (rnbId: string) => {
+    dispatch(
+      Actions.edition.setCandidates(formatCandidates(rnbId, candidatesToMerge)),
+    );
+  };
+  const handleSubmit = async () => {
+    setIsLoading(true);
+    const url = `${process.env.NEXT_PUBLIC_API_BASE}/buildings/merge/`;
+
+    const data: { [key: string]: any } = {
+      rnb_ids: candidatesToMerge,
+      status: 'constructed',
+      merge_existing_addresses: true,
+    };
+
+    try {
+      const response = await fetch(url, {
+        body: JSON.stringify(data),
+        method: 'POST',
+      });
+
+      if (!response.ok) {
+        await throwErrorMessageForHumans(response);
+      } else {
+        // force the map to reload the building, to immediatly show the modifications made
+        dispatch(Actions.map.reloadBuildings());
+        const data = await response.json();
+        dispatch(Actions.map.selectBuilding(data.rnb_id));
+        toasterSuccess(dispatch, `Les bâtiments ont été fusionnés avec succès`);
+        setNewBuilding(data);
+      }
+      setIsLoading(false);
+    } catch (err: any) {
+      toasterError(dispatch, err.message || 'Erreur lors de la fusion');
+      setIsLoading(false);
+      console.error(err);
+    }
+  };
+  return (
+    <>
+      <RNBIDHeader>
+        <h1 className="fr-text--lg fr-m-0">Fusionner des bâtiments</h1>
+      </RNBIDHeader>
+      {newBuilding?.rnb_id ? (
+        <div className={styles.mergePanel_body}>
+          <MergeSummary
+            newBuilding={newBuilding}
+            buildingsMerged={candidatesWithAddresses}
+          />
+          <div className={styles.footer}>
+            <Button onClick={newMerge}>Nouvelle fusion</Button>
+          </div>
+        </div>
+      ) : (
+        <div className={styles.mergePanel_body}>
+          <div className={styles.mergePanel__descWrapper}>
+            <span className={styles.mergePanel__descText}>
+              Sélectionner les bâtiments à fusionner
+            </span>
+            <span className={styles.mergePanel__descSubText}>
+              Je sélectionne sur la carte les bâtiments attenants qui doivent
+              être fusionner
```

Comment:
```suggestion
              Sélectionnez sur la carte les bâtiments attenants qui doivent
              être fusionnés
```

Dans le panneau create on ne parle pas à la première personne (_Tracez la géométrie du bâtiment sur la carte._)